src/api/app/views/webui2/webui/project/bottom_actions/_modify_project.html.haml

```diff
@@ -0,0 +1,12 @@
+- unless project.defines_remote_instance?
+  - if is_incident_project && packages.present? && has_patchinfo && open_release_requests.blank?
```

Comment:
I didn't check how the old code does this, but if the project has a patchinfo, we should also show it.

Reply:
The patchinfo is displayed, but only if we get to the last `else`. I followed what we had in the old view.

Reply:
That's what I meant. Why would we hide the patchinfo for some projects? :thinking: But if it was like that before, let's keep it for now.

Reply:
Alright. I'm not sure too why it would be hidden to be honest